lambdas/generator/handler.py

```diff
@@ -7,21 +7,54 @@
 from dotenv import load_dotenv
 from openai import OpenAI
 import requests
+import uuid
+import boto3
 
 
+# Load environment variables
```

Comment:
I think we can use ssm parameters for this instead of loading .env files for security reasons

Reply:
I will create a PR for it. Good for now.